src/patternfly/components/Tabs/tabs.scss

```
@@ -688,6 +674,19 @@ $pf-c-tabs--spacer-map: build-spacer-map("none", "sm", "md", "lg", "xl", "2xl");
     --pf-c-tabs__link-toggle-icon--Color: var(--pf-c-tabs__link--m-expanded__toggle-icon--Color);
     --pf-c-tabs__link-toggle-icon--Rotate: var(--pf-c-tabs__link--m-expanded__toggle-icon--Rotate);
   }
+
+  &:hover {
+    --pf-c-tabs__link-toggle-icon--Color: var(--pf-c-tabs__link--hover--Color);
+  }
+
+  &:focus {
+    --pf-c-tabs__link-toggle-icon--Color: var(--pf-c-tabs__link--focus--Color);
+  }
+
+  &:active,
+  &.pf-m-active {
+    --pf-c-tabs__link-toggle-icon--Color: var(--pf-c-tabs__link--active--Color);
+  }
```

Could you move this up so it comes after the existing hover/focus/active state styles? https://github.com/patternfly/patternfly/blob/71925cdb08028f786a060befdfcdfb4585a7de32/src/patternfly/components/Tabs/tabs.scss#L622-L640

You could just add these styles to that block, though it will add them to the `.pf-m-action` items, too, and I don't think it needs to be there since you wouldn't want to trigger this color change if hovered over some other action in a tab item (like the close button).

Could you also rename the vars so it's

```suggestion
  &:hover {
    --pf-c-tabs__link-toggle-icon--Color: var(--pf-c-tabs__link--hover__toggle-icon--Color);
  }

  &:focus {
    --pf-c-tabs__link-toggle-icon--Color: var(--pf-c-tabs__link--focus__toggle-icon--Color);
  }

  &:active,
  &.pf-m-active {
    --pf-c-tabs__link-toggle-icon--Color: var(--pf-c-tabs__link--active__toggle-icon--Color);
  }
```